README.md

```diff
@@ -45,6 +45,11 @@ When at the `web` directory, run the following:
 - Change `dbuser@127.0.0.1/db_name` with your DB settings
 - The last bit - `idea` is the name of the profile
 
+## Update
+
+1. Run `composer update`
```

Comment:
`composer update` will update all dependencies, I don't think this is necessary.
Should it be something like `composer update openideal/openideal --with-dependencies`?

Reply:
Done